packages/evm/src/opcodes/gas.ts

```diff
@@ -476,8 +489,25 @@ export const dynamicGasHandlers: Map<number, AsyncDynamicGasHandler | SyncDynami
         }
         gas += subMemUsage(runState, inOffset, inLength, common)
         gas += subMemUsage(runState, outOffset, outLength, common)
+
+        let charge2929Gas = true
+        if (common.isActivatedEIP(6800)) {
+          // TODO: add check if toAddress is not a precompile
```

Comment:
Should this TODO be resolved before we merge?  Seems like a relatively simple check.

Reply:
need to do a precomiple check at lot many places, will do a followup PR for the same if thats ok